src/mesh/mesh.cpp

```diff
@@ -75,12 +75,12 @@ Mesh::Mesh(ParameterInput *pin, ApplicationInput *app_in, Properties_t &properti
                 pin->GetInteger("parthenon/mesh", "nx2"),
                 pin->GetInteger("parthenon/mesh", "nx3")},
       mesh_bcs{
-          GetBoundaryFlag(pin->GetOrAddString("parthenon/mesh", "ix1_bc", "reflecting")),
```

Comment:
I'd rather not change these defaults to periodic.

Reply:
I'll change it back. But I would be curious as to why.

Reply:
changed.